### PR TITLE
Audit log streaming

### PR DESCRIPTION
**Pull request by bennettgo**
### PR submission checklist
- [x] Optimised every new image
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to SUMMARY.md

### CODE REVIEW

**Comment by bennettgo** [comment]
done!

File: job-history-replication.md [hunk 1]
```diff
@@ -1,104 +1,1482 @@
...
+
+        }
+      }
+    },
+    "csv_parser":{
+      "4e1c94fb":{
+        "lines":[
+          {
+            "column_0":"Sierra Gardening",
+            "column_1":" MS-321",
```
Comment (eeshansim) [inline job-history-replication.md]: I think you accidentally added the space after `"` for these lines

**Comment by eeshansim** [comment]
@bennettgo this is not resolved yet https://github.com/workato/docs/pull/915#discussion_r338902567

**Comment by eeshansim** [comment]
Issue with right-sidebar
image

File: job-history-replication.md (hunk 1 above)
Comment (bennettgo) [reply job-history-replication.md]: fixed

**Pull request closed without merging**